hack/local-up-cluster.sh

```diff
@@ -216,18 +249,24 @@ function start_apiserver {
     if [[ -n "${RUNTIME_CONFIG}" ]]; then
       runtime_config="--runtime-config=${RUNTIME_CONFIG}"
     fi
-
     APISERVER_LOG=/tmp/kube-apiserver.log
-    sudo -E "${GO_OUT}/kube-apiserver" ${priv_arg} ${runtime_config}\
+
+    # We provide a token auth, so that kubelets can access the api-server
+    # through a service endpoint.  This allows for highly available api-server
+    # setups.
+    sudo -E "${GO_OUT}/kube-apiserver" ${priv_arg} ${runtime_config} \
```

Comment:
local-up-cluster seems like a weird place to introduce HA master component concepts. I'd vastly prefer a path that is exercised by merge queue e2e tests.